ocs_ci/helpers/helpers.py

```diff
@@ -5630,3 +5630,158 @@ def apply_custom_taint_and_toleration(taint_label="xyz"):
             )
             for pod_obj in pod_list:
                 pod_obj.delete(wait=False)
+
+
+def create_ceph_block_pool_for_deviceclass(
```

Comment:
I suggest trying to integrate deviceclass into the existing helper function of create_ceph_block_pool() - https://github.com/red-hat-storage/ocs-ci/blob/3d8e5d8f68210023671c50e10f0ba3b81bca3035/ocs_ci/helpers/helpers.py#L576C5-L576C27

This will prevent duplicity

Reply:
Yes, I thought of that, but the Yaml files have some diffs.

Reply:
Fixed

Reply:
I mean, I fixed it locally. Now, I pushed the changes.